backend/prompt_studio/prompt_studio/constants.py

```diff
@@ -13,6 +13,10 @@ class ToolStudioPromptKeys:
     PROMPT_KEY = "prompt_key"
     EVAL_METRIC_PREFIX = "eval_"
     EVAL_RESULT_DELIM = "__"
+    SEQUENCE_NUMBER = "sequence_number"
+    START_SEQUENCE_NUMBER = "start_sequence_number"
+    DROP_SEQUENCE_NUMBER = "drop_sequence_number"
```

Comment:
NIT: Consider naming it `end_sequence_number` to compliment the naming of `start_sequence_number`

Reply:
@chandrasekharan-zipstack Thanks for the suggestion. Done.